### PR TITLE
error_details

### PR DESCRIPTION
**Pull request by RomanPudashkin**
Resolves: https://github.com/musescore/MuseScore/issues/15226

### CODE REVIEW

**Comment by bkunda** [comment]
@RomanPudashkin I've tested this on macOS using all your supplied scores, as well as the "Avengers" score we were previously using. 
✅ Both "types" of error dialog are being triggered correctly (I.e. the "open anyway" version, and the "get help" version)
✅ "Get help" link points to correct URL
✅ "Show details" text strings are correctly laid-out
✅ "Copy" functionality works
Let us know when the accessibility navigation is implemented and I'll give that one quick look too 🙂

File: src/framework/ui/view/internal/errordetailsmodel.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "errordetailsmodel.h"
+
+#include <QApplication>
+#include <QClipboard>
+#include <QTextDocument>
+
+#include "translation.h"
```
Comment (Eism) [inline src/framework/ui/view/internal/errordetailsmodel.cpp:29]: don't need

**Review by Eism — APPROVED** [review]

**Comment by Jojo-Schmitz** [comment]
Seems this PR needs thee "Strings" label, as it adds translatable strings?
And wouldn't that make it not fitting 4.0.1?

**Comment by bkunda** [comment]
@RomanPudashkin I've given this another test on macOS – all looks good (and keyboard navigation works well too).
Just one quibbly point: could we please have a space after the punctuation marks at these points? 

Screenshot 2023-01-18 at 11 42 09 am

**Comment by cbjeukendrup** [comment]
@bkunda That's the same problem again as #14560...

**Comment by bkunda** [comment]
> @bkunda That's the same problem again as #14560... (where the only solution is to update to Qt 5.15.3+)

oh god, how droll. 
Ok then @RomanPudashkin this is good to go from my POV!

**Pull request merged**